Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift

```diff
@@ -174,6 +195,19 @@ internal class IOrchestratedDocumentVerificationViewModel<T, U: JobResult>: Obse
                 receiveCompletion: { completion in
                     switch completion {
                     case .failure(let error):
+                            do {
```

Comment:
Can you add a TODO here to integrate with the SmileID.allowOfflineMode property?

Reply:
done